### PR TITLE
Add new recipe for ANTs

### PR DESCRIPTION
**Pull request by IanButterworth**
I completed the wizard recipe by adding the script after terminating during the first platform build. I didn't finish running the first platform as it was taking a long time

### CODE REVIEW

**Comment by ianshmean** [comment]
Windows should be possible: https://github.com/ANTsX/ANTs/wiki/Compiling-ANTs-on-Windows-10

File: A/ANTs/build_tarballs.jl [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+# Collection of sources required to complete build
+sources = [
+    GitSource("git://github.com/stnava/ANTs.git", "857b744cc6eeb72d5cd388933a3492846274cfdf")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+mkdir build install
+cd build
+cmake ../ANTs -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release
```
Comment (giordano) [inline A/ANTs/build_tarballs.jl:17]:
```suggestion
cd ANTs
mkdir build && cd build
cmake -DCMAKE_INSTALL_PREFIX=${prefix} \
    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
    -DCMAKE_BUILD_TYPE=Release \
    ..
```

**Comment by ianshmean** [comment]
I can't figure out where `-soname` is being introduced (instead of `-install_name`) on darwin https://dev.azure.com/JuliaPackaging/Yggdrasil/_build/results?buildId=6301&view=logs&j=a7ccc0f4-5533-5774-a96d-9e5713500d94&t=6d73f7b1-4cfd-512e-4a29-6889443db5c1&l=3205

@giordano any ideas?

**Comment by imciner2** [comment]
Closing due to inactivity. Please submit a new PR on top of the current master branch if you still want to include this recipe.

**Pull request closed without merging**